backend/FwLite/LcmCrdt/Changes/Entries/ReplaceComplexFormTypeChange.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MiniLcm.Models;
+using SIL.Harmony.Changes;
+using SIL.Harmony.Entities;
+
+namespace LcmCrdt.Changes.Entries;
+
+public class ReplaceComplexFormTypeChange(Guid entityId, ComplexFormType newComplexFormType, Guid oldComplexFormTypeId) : EditChange<Entry>(entityId), ISelfNamedType<ReplaceComplexFormTypeChange>
+{
+    public ComplexFormType NewComplexFormType { get; } = newComplexFormType;
+    public Guid OldComplexFormTypeId { get; } = oldComplexFormTypeId;
+
+    public override ValueTask ApplyChange(Entry entity, ChangeContext context)
+    {
+        entity.ComplexFormTypes =
+        [
+            ..entity.ComplexFormTypes.Where(t => t.Id != OldComplexFormTypeId),
```

Comment:
The current plan is to eventually use a sort-order field for persisting sorting, right?

Reply:
Yes, I've made a todo comment on the ComplexFormType class